SlashCommandsExample/Program.cs

```diff
@@ -0,0 +1,26 @@
+/*
+ * This project, is at this moment used for testing and debugging the new and experimental Slash Commands.
+ * After all testing has been done, and the project is ready to be integrated into the main Discord.Net ecosystem
+ * this project should be re-made into one that could be used as an example usage of the new Slash Command Service.
+ */
+using System;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+using Discord.SlashCommands;
+using Discord.WebSocket;
+
+namespace SlashCommandsExample
+{
+	class Program
+	{
+        static void Main(string[] args)
```

Comment:
Nit: You can declare `Main` as `Task Main(string[] args)` and simply `await` your `RunAsync()` call to avoid mixing synchronous and asynchronous code.

Reply:
It doesn't *really* matter here, as they both ultimately compile down to the same IL internally; at least, as far as this example is concerned.